citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -334,6 +409,42 @@ <h4 class="panel-title">
 		</tr>
 	</table>
 </div>
+
+<div class="modal fade" id="entry-delete-confirmation" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">Delete Entry</h4>
+      </div>
+      <div class="modal-body">
+        <p>Are you sure you want to delete the selected Entry</p>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">No, cancel!</button>
+        <button id="delete-entry-button" data-entry-key="" type="button" class="btn btn-primary">Yes, delete!</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal fade" id="no-selected-entry" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">Delete Entry</h4>
+      </div>
+      <div class="modal-body">
+        <p>Please select at least 1 Entry !!!</p>
```

Comment:
This doesn't need three "!", there shouldn't be a blank between Entry and "!", and "Entry" needs to be lowercased.